libmscore/paste.cpp

```diff
@@ -562,13 +562,30 @@ void Score::pasteChordRest(ChordRest* cr, const Fraction& t, const Interval& src
       {
       Fraction tick(t);
 // qDebug("pasteChordRest %s at %d, len %d/%d", cr->name(), tick, cr->ticks().numerator(), cr->ticks().denominator() );
-      if (cr->isChord())
-            transposeChord(toChord(cr), srcTranspose, tick);
 
       Measure* measure = tick2measure(tick);
       if (!measure)
             return;
 
+      int twoNoteTremoloFactor = 1;
+      if (cr->isChord()) {
+            transposeChord(toChord(cr), srcTranspose, tick);
+            if (toChord(cr)->tremolo() && toChord(cr)->tremolo()->twoNotes())
+                  twoNoteTremoloFactor = 2;
+            else if (cr->durationTypeTicks() == (cr->actualTicks() * 2)) {
+                  // this could be the 2nd note of a two-note tremolo
+                  // check previous CR on same track, if it has a two-note tremolo, then set twoNoteTremoloFactor to 2
+                  Segment* seg = measure->undoGetSegment(SegmentType::ChordRest, tick);
+                  ChordRest* crt = seg->nextChordRest(cr->track(), true);
+                  if (crt && crt->isChord()) {
+                        Chord* chrt = toChord(crt);
+                        Tremolo* tr = chrt->tremolo();
+                        if (tr && tr->twoNotes())
```

Comment:
The previous chord could be also the second chord for its tremolo, would it make sense to add a check for something like `tr->chord1() == chrt`?

Reply:
True, but I don't know if the tremolo have already been fixed at the time the check is made (indeed the second note of a two-note tremolo reports a 0 tremolo), but much better safe than sorry. I will add this check and an mtest for this case this evening.
Thank you. 😄

Reply:
I found that the two-note tremolo is not yet completed at this point, and _both_ chord1() and chord2() are still 0 for the tremolo tr.
So such a check would lead to a wrong result in this case.

Reply:
So I just rebased the PR without changes.